Makefile

```
@@ -43,6 +43,12 @@ clean-db:  ## remove LMS databases
 	$(COMPOSE) rm edx_mongodb edx_mysql edx_redis 
 .PHONY: clean-db
 
+e2e-tests:\
+	run
+e2e-tests: ## run end-to-end tests
```

I would remove the `run` rule dependency as it's a bit long to execute each time you want to test your app, and `test` is faster to type!

```suggestion
test: ## run end-to-end tests
```